trace/trace_test.go

```diff
@@ -17,14 +17,17 @@ package trace
 import (
 	"context"
 	"fmt"
+	"go.opencensus.io/trace/tracestate"
```

Comment:
Minor, put custom (go.opencensus.io) packages below the standard packages, e.g.:

```
import (
	"log"
	"net/http"

	"go.opencensus.io/plugin/ochttp"
	"go.opencensus.io/plugin/ochttp/propagation/b3"
	"go.opencensus.io/stats/view"
	"go.opencensus.io/tag"
)
```

Reply:
+1

Reply:
fixed.